content/swift/concepts/optionals/optionals.md

```diff
@@ -0,0 +1,94 @@
+---
+Title: 'Optionals'
+Description: 'Optionals are types that deal with scenarios involving the potential presence or absence of a value (nil).'
+Subjects:
+  - 'iOS'
+  - 'Mobile Development'
+Tags:
+  - 'Optionals'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+Optionals are types that deal with scenarios involving the potential presence or absence of a value (nil).
+
+## Syntax
+
+In Swift, a non-optional variable can't be set to nil. Rather, the optional type must be defined using the added question mark `?`.
+
+Below, a variable `myInt` is declared and set to the optional `Int` type, `Int?`. It can either be equal to any integer or nil.
+
+```swift
+var maybeInt: Int? = nil
+```
+
+**Note:** If `myInt` is not assigned the value of nil, it will default to nil.
+
+## Unwrapping Optionals
+
+Optional types can be unwrapped if it's certain that a value is present. Below are some ways this can be done.
+
+### Nil Coalescing
+
+The nil coalescing operator, `??`, checks the value on the left. If that value is nil, the default value on the right side is used.
```

Comment:
```suggestion
The nil-coalescing operator, `??`, is used to assign a value by checking from left to right and assigns a default value if all the preceding `Optional` values were `nil`. Per the example below, if `maybeInt` is `nil`, the default value on the right side, `27`, will be assigned to `myInt`:
```

Reply:
Thanks for helping to make this clearer! How does this compare:

> The nil-coalescing operator, `??`, assigns a value by checking from left to right. If the preceding `Optional` values are `nil`, the default value on the right is assigned. In the example below, since `maybeInt` was previously declared as `nil`, `myInt` will default to 27:

Reply:
This is awesome!